featurebyte/worker/task_executor.py

```diff
@@ -130,6 +131,12 @@ def _setup_worker_config(self) -> None:
             "default_profile: worker\n\n",
             encoding="utf-8",
         )
+        # NOTE: Unable to remove this try-except block
+        # due to test_task_manager expecting a server listening on 127.0.0.1:8080/status endpoint
```

Comment:
Would it be possible to patch test_task_manager to avoid this issue? Seems a bit risky to mask an exception here when that is not expected in actual code paths.